### PR TITLE
Hotfix validate schedule

### PR DESCRIPTION
**Pull request by MustafaWasif**
Updated processing data from FE (when editing schedule) for the following:
- `startDate`
- `endDate`
- `beginTime`
- `endTime`
- `displayName`

### CODE REVIEW

File: src/resolvers/resolverUtils.ts [hunk 1]
```diff
@@ -347,15 +351,20 @@ export function prepareCourseCapacities({
   return combinedRequest;
 }
 
-function courseSectionInputToCourse(course: CourseSectionInput) {
+async function courseSectionInputToCourse(
+  course: CourseSectionInput
+): Promise<CourseType> {
+  const user = await findUserByUsername(course.professors[0]);
```
Comment (aqeelmozumder) [inline src/resolvers/resolverUtils.ts:357]: this will only grab the first professor? is it because the professor field has 1 user only?

**Review by aqeelmozumder — COMMENTED** [review]
Ran and Tested the Code, Works Expected.

**Review by aqeelmozumder — APPROVED** [review]
LGTM

File: src/resolvers/resolverUtils.ts (hunk 1 above)
Comment (MustafaWasif) [reply src/resolvers/resolverUtils.ts:357]: Yes, the professor field is presumed to have 1 user only. It was discussed earlier with FE (I think Ken raised that topic).

Comment (MustafaWasif) [reply src/resolvers/resolverUtils.ts:357]: https://discord.com/channels/973344930699313203/973345629914947675/998728216825045023

**Pull request merged**